core/src/main/java/org/apache/brooklyn/core/mgmt/internal/LocalLocationManager.java

```diff
@@ -78,9 +85,13 @@
     public LocalLocationManager(LocalManagementContext managementContext) {
         this.managementContext = checkNotNull(managementContext, "managementContext");
         this.locationFactory = new InternalLocationFactory(managementContext);
-        
+
         this.storage = managementContext.getStorage();
-        locationTypes = storage.getMap("locations");
+        this.locationTypes = storage.getMap("locations");
+
+        BasicExecutionManager executionManager = new BasicExecutionManager(managementContext.getManagementNodeId());
```

Comment:
Do we really need to create our own `BasicExecutionManager`? Can we not reuse `managementContext.getExecutionManager()`?